source/api/tests/test_api_unit_test.py

```diff
@@ -0,0 +1,292 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# ###############################################################################
+# PURPOSE:
+#   * Unit test for project api endpoints and workflow.
+#
+# USAGE:
+#   ./run_api_test.sh --run_unit_test
```

Comment:
is this correct usage?

Reply:
But the filename is different.

Reply:
i can rename `--run_unit_test` to `test_api_unit_test`, if you prefer that.

Reply:
Under USAGE, you need to indicate which directory they need to be in to run the script. You reference ./, but run_api_test.sh is not in ./.

Also, is there are reason why run-unit-tests.sh is in the deployment directory? If it's a deployment asset, that would make sense, but it's a test script, so I think it makes more sense for it to be in source/tests/.

Reply:
`run-unit-tests.sh` was always in the deployment dir.

This script is just helper to run the test within the test directory/folder.

Reply:
I have resolved the Usage verbiage.